### PR TITLE
Initial implementation of first community board

### PR DESCRIPTION
**Pull request by piwi1263**
This is the first community board for an NUCLEO-F411RE with the ST-LINK board cut off. 

As a result of the missing ST_Link part we have no external Xtal anymore.
Additional components have been added to the board to have an HSE again.
How to achieve this can all be read in the UM1724 manual on page 24.
The manual can be found on http://www.st.com and search for NUCLEO-F411RE.

The power supply is gone also, so I have made a USB cable to connect PA11->DM, PA12->DP and use the USB power wires too.
Don't forget to set jumper JP5 to use E5V and put Boot0 to Vss to put the device in DFU mode so you can burn the binary images. I use the STDFUTester application of ST to erase and burn the images.

Signed-off-by: Peter Wessel <piwi1263@gmail.com>

### CODE REVIEW

File: CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Definitions for each image ## 
+
+Any defines specific to an image (nanoBooter or nanoCLR) that are to be made available throughout the code should be added to the _target_board.h.in_. There is one in [nanoBooter](nanoBooter/target_board.h.in) and another in [nanoCLR](nanoCLR/target_board.h.in) directories, allowing different defines for each one.
+Defines that are common to a target board should be added to the [_target_common.h.in_](target_common.h.in).
+The content of these files is parsed by CMake. It can include CMake variables. Check the documentation of CMake config files [here](https://cmake.org/cmake/help/v3.7/command/configure_file.html?highlight=configure_file).
+
+
+
+
+**NOTE: this configuration was successfully tested in a NUCLEO_F091RC board using the virtual COM port through the ST Link USB connection.**
```
Comment (josesimoes) [inline CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md]: I'm probably being picky... but... could you please correct the board name and the description on the connection? I'm under the impression that you are not using the ST Link virtual COM port... 😄

File: CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+## Configuration of Chibios, HAL and MCU ##
+
+For a successful build the following changes are required:
+
+In _halconf.g_ (in both nanoBooter and nanoCLR folders), when compared with a default file:
```
Comment (josesimoes) [inline CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md]: I believe that this is not accurate... see bellow please.

Comment (piwi1263) [inline CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md]: More accurate text put in place.

File: CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md (hunk 1 above)
Comment (piwi1263) [inline CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md]: Yes you are but that doesn't change the fact it is more accurate as it is with the latest readme.md inside the board folder. I hope it suffices for the moment.

**Pull request merged**